hnn_core/network.py

```diff
@@ -764,6 +793,254 @@ def _get_src_type_and_pos(self, gid):
 
         return src_type, src_pos, src_type in self.cellname_list
 
+    # connections:
+    # this NODE is aware of its cells as targets
+    # for each syn, return list of source GIDs.
+    # for each item in the list, do a:
+    # nc = pc.gid_connect(source_gid, target_syn), weight,delay
+    # Both for synapses AND for external inputs
+    def _default_connect(self):
```

Comment:
I like that the default network connections are now getting defined here instead of `NetworkBuilder`. +1 for all python-defined network properties being defined in `Network`!

What do you think of naming this method `_set_default_connections`? We've now introduced a bunch of connectivity-related methods to `Network` and I'm worried that new users will find it hard to follow which function does what.